Ontology/Willow/Component/Asset/Fan.json

```diff
@@ -155,6 +165,37 @@
       "schema": "VolumeFlowRateUnit",
       "writable": true
     },
+    {
+      "@type": ["Property", "VolumeFlowRate"],
+      "displayName": {
+        "en": "Design Airflow"
+      },
+      "name": "designAirflow",
+      "schema": "double",
+      "unit": "litrePerSecond",
```

Comment:
The units are all metric, will that matter when it comes to rules engine? Is there a way, in the load sheets, to make the units for asset data site specific?

Reply:
These are just default units which DTDL requires us to define in the model, but we always have a sister property with the -Unit suffix where we can define the actual twin's units. For this property, that is shown in lines 179-187.